lib/provider/tests/test_views.py

```diff
@@ -28,9 +28,11 @@ class TestAPIView(TestCase):
 
     def test_no_reference(self):
         req = RequestFactory().get('/')
-        with self.settings(ZIPPY_MOCK=False):
-            eq_(ProxyView().dispatch(req, reference_name='bob',
-                                     resource_name='sellers').status_code, 404)
+        with self.assertRaises(KeyError):
```

Comment:
what is this test doing? Do we still need it? I can't decipher from the test name what it's testing.